projects/components/src/form/number-with-unit-input/number-with-unit-form-input.component.ts

```diff
@@ -333,6 +337,15 @@ export class NumberWithUnitFormInputComponent extends BaseFormControl implements
         }
     }
 
+    /**
+     * Checks if the value in the form is the unlimited value.
```

Comment:
It is confusing:
`Checks if the value in the form is the unlimited value` this means the method should return `true` or `false`